src/pyhf/utils.py

```diff
@@ -274,18 +351,21 @@ def hypotest(
 
     init_pars = init_pars or pdf.config.suggested_init()
     par_bounds = par_bounds or pdf.config.suggested_bounds()
+    test_stat = qmu if not use_q0 else q0
+    if qtilde and use_q0:
```

Comment:
probably this should become a better look up generally now that we expand the number of test statistics
```
test_stats = {
    'q_mu': ..
    'q_tilde_mu': ..
    'q_0': ..
}
```

Reply:
This is a good idea, but there are probably more places where a distinction has to be made. For example the way how to calculate p-values from the test statistic (mainly different between q_mu and q_tilde) and if one wants cls or pvalues (cls doesn't really make sense for q_0)

Reply:
I think the best way for now is to keep it as it is, and then clean it up after this PR.